src/DotLiquid.Tests/TemplateTests.cs

```diff
@@ -8,47 +8,56 @@ namespace DotLiquid.Tests
     [TestFixture]
     public class TemplateTests
     {
+
+        private System.Collections.Generic.List<string> Tokenize(string input)
+        {
+            var v20 = Tokenizer.Tokenize(input, SyntaxCompatibility.DotLiquid20);

```

Comment:
Why 2.0 not 2.1 to compare with 2.2?

Reply:
2.0 and 2.1 have no difference in parsing step (Tokenizer.Tokenize is part of Template.Parse). 2.2 introduces the whitespace control handling. Comparing earliest, the default shipped with DotLiquid, to latest probably makes the most sense.